Resources/Prototypes/_NF/Shipyard/stellaris.yml

```diff
@@ -0,0 +1,30 @@
+- type: vessel
+  id: stellaris
+  name: NT Stellaris
+  description: A mobile theatre perfect for putting on any show.
+  price: 30000
+  category: Medium
+  group: Civilian
+  shuttlePath: /Maps/Shuttles/stellaris.yml
+
+- type: gameMap
+  id: Stellaris
```

Comment:
you have something mismatched on your prototype file and its causing the ship to not spawn correctly. Make sure all of your IDs match and that the becomesStation component field matches the entries here.